### DISCUSSION TITLE
Documenting on how to set up this project on Intellij IDEA?

### DISCUSSION THREAD

**Discussion by mio-19** [aya-prover/aya-dev#1133] [General]
Opening this project on Intellij IDEA without any configuration gives me a ton of errors and I forgot how to fix this. How about make a document about how to set up this project on Intellij IDEA?
<img width="2352" alt="Screenshot 2024-08-05 at 3 02 36 PM" src="https://github.com/user-attachments/assets/c46ebbc4-0738-4464-8b40-659634450824">

**Comment by mio-19** [comment]
<img width="2352" alt="Screenshot 2024-08-05 at 3 04 48 PM" src="https://github.com/user-attachments/assets/c4d2a1cf-8cf8-4f2f-b29a-84b7d01f1333">

**Comment by imkiva** [comment]
I also forgot how to make IDEA happy 🤣

**Comment by ice1000** [comment]
Interesting, can you show me your IDEA version and JDK version?

**Reply by ice1000** [reply]
If it's a gradle error, it might be that you just need to wait a bit more for it to load the project

**Comment by mio-19** [comment]
idea is happy now after I reconfigured project SDK and reloaded gradle project
